package.json

```diff
@@ -26,6 +26,12 @@
       "title": "Try using the local tslint package (if exist)",
       "default": true
     },
+    "enableSemanticRules": {
+      "type": "boolean",
+      "title": "Enable semantic rules",
+      "description": "Pass a TypeScript program object to the linter. May negatively affect performance. See this page for details: https://palantir.github.io/tslint/usage/type-checking/",
```

Comment:
"Allow passing a..."

This should be conditionally worded since it doesn't forcibly pass one every time.